lms/djangoapps/grades/tests/test_events.py

```diff
@@ -0,0 +1,94 @@
+"""
+Test that various events are fired for models in the grades app.
+"""
+
+from unittest import mock
+from django.utils.timezone import now
+
+from lms.djangoapps.grades.models import PersistentCourseGrade
+
+from openedx_events.learning.data import PersistentCourseGradeData, CourseData # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
why do we need the `disable=wrong-import-order` here?

This ordering should be fixed instead.

Reply:
thanks a lot, I have already fixed the order of the imports.